packages/rsocket-messaging/src/DefaultResponders.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  Cancellable,
+  FrameTypes,
+  OnExtensionSubscriber,
+  OnNextSubscriber,
+  OnTerminalSubscriber,
+  Payload,
+  Requestable,
+} from "@rsocket/core";
+import { Codec } from "./index";
+
+export type DefaultResponderHandlerSignature<T> = (
+  p: T,
+  r: number,
+  s: OnTerminalSubscriber & OnNextSubscriber & OnExtensionSubscriber
+) => Cancellable & OnExtensionSubscriber & Requestable;
+
+export function requestStream<T, R>(
+  handler: DefaultResponderHandlerSignature<T>,
+  codecs: {
+    inputCodec: Codec<T>;
+    outputCodec: Codec<R>;
+  }
+) {
+  return Object.assign<
+    DefaultResponderHandlerSignature<Payload>,
+    { requestType: FrameTypes.REQUEST_STREAM }
+  >(
+    (payload, initialRequestN, subscriber) => {
+      return handler(
+        codecs.inputCodec.decode(payload.data),
+        initialRequestN,
+        subscriber
+      );
+    },
+    { requestType: FrameTypes.REQUEST_STREAM }
+  );
+}
```

Comment:
As I mentioned in the last PR, I started adding a set of "default" responders to aid with my testing, as well as an additional feature. I only got as far as this single responder though, so if we don't want to land this with only a single responder, we can pick it out and loop back around to adding the rest later.

Reply:
There is also the question of if these default request/responders should be in the messaging package or elsewhere.